tests/features/environment.py

```diff
@@ -134,6 +134,7 @@ def after_scenario(context, _):
                 '{0}> '.format(dbname),
                 timeout=5
             )
+        context.cli.sendcontrol('u')
```

Comment:
for mycli I had to use `sendcontrol('c')` to have always a "clean" prompt... ctrl-u removed the current line and there might be a multi-line statement in the prompt...

Reply:
Well, after scenario is done, there shouldn't be anything in prompt, that's the point.

Reply:
while using ctrl-u there might be something left at the prompt though...

Reply:
Didn't know that. In that case I better switch to Ctrl + C.